docs/supported.md

```diff
@@ -40,7 +41,7 @@ Earlier and Later versions might be only partially supported, we don't fully kno
 
 ## next.config.js Properties
 
-> The following options have been gathered from the [Next.js next.config.js documentation](https://nextjs.org/docs/api-reference/next.config.js/introduction).
+> The following options have been gathered from the Next.js [next.config.js documentation](https://nextjs.org/docs/api-reference/next.config.js/introduction), alongside the [Advanced routing documentation](https://nextjs.org/docs/advanced-features/i18n-routing).
```

Comment:
sorry my bad :bow: :pray: 
```suggestion
> The following options have been gathered from the Next.js [next.config.js documentation](https://nextjs.org/docs/api-reference/next.config.js/introduction), alongside the [Internationalized routing documentation](https://nextjs.org/docs/advanced-features/i18n-routing).
```

Reply:
technically they would both be okay :p

Reply:
yeah... :thinking: :sweat_smile: 

(but I guess using their same wording is better :sweat_smile: :grin:)